### ISSUE TITLE
Accuracy of CTRGCN cannot be reproduced

### ISSUE THREAD

**Issue by LostDawn** [kennymckormick/pyskl#26]
I try to train CTRGCN with 2d skeleton and joint modality, but the accuracy dropped over 1%. I didn't change anything, except training with only 4 gpus. Would a few less gpus have such a big impact on accuracy? It's quite strange. So I wonder if you have tuned any hyperparameters during training. Thanks in advance.

**Comment by kennymckormick** [comment]
If you use 4GPUs, you should also decrease the learning rate by half. We use linear scaling rule.

**Comment by LostDawn** [comment]
But DDP average gradients from all gpus, so it seems gpu number doesn't affect equvalent batch size?

**Comment by kennymckormick** [comment]
Yep, it's not about the magnitude of gradient, but about the variance of the gradient. Larger batch size -> smaller gradient variance -> Larger LR.

**Comment by LostDawn** [comment]
Well, I tried to train with 8 gpus, and the result is comparable to reported. However, use 4 gpus with half of default lr cannot get the same accuracy. Thanks.
